tools/klee/main.cpp

```diff
@@ -1227,6 +1244,113 @@ void KleeHandler::dumpCallPath(const ExecutionState &state,
+  std::vector<std::string> dpdk_fns = {"rte_reset", "rte_arch_bswap16", "rte_arch_bswap32", "rte_arch_bswap64", "__rte_raw_cksum", "__rte_raw_cksum_reduce", "rte_raw_cksum", "rte_ipv4_phdr_cksum", "rte_ipv4_cksum", "rte_ipv4_udptcp_cksum", "rte_exit", "rte_lcore_is_enabled", "rte_get_master_lcore", "rte_get_closest_next_lcore", "rte_eth_tx_burst", "flood", "rte_pktmbuf_free", "rte_get_tsc_hz", "rte_lcore_id", "rte_rdtsc", "rte_eth_rx_burst", "rte_prefetch0", "rte_lcore_is_enabled", "rte_lcore_to_socket_id", "rte_socket_id", "rte_eth_dev_socket_id", "rte_eth_link_get_nowait", "rte_delay_ms", "rte_eal_init", "rte_eth_dev_count", "rte_lcore_count", "rte_eth_dev_configure", "rte_eth_macaddr_get", "rte_eth_dev_info_get", "rte_eth_tx_queue_setup", "rte_eth_rx_queue_setup", "rte_eth_dev_start", "rte_eth_promiscuous_enable", "rte_eal_mp_remote_launch", "rte_eal_wait_lcore", "rte_pktmbuf_pool_create", "rte_get_master_lcore", "rte_strerror", "rte_pktmbuf_clone", "cmdline_isendoftoken", "nf_set_ipv4_checksum"};
+  std::vector<std::string> time_fns = {"start_time", "restart_time", "current_time", "get_start_time_internal", "get_start_time", "clock_gettime", "gettimeofday"};
+  std::vector<std::string> verif_fns = {"loop_iteration_assumptions", "loop_iteration_assertions", "loop_invariant_consume", "loop_invariant_produce", "loop_iteration_begin", "loop_iteration_end", "loop_enumeration_begin", "loop_enumeration_end", "allocate_unique_name", "count_reuse", "init_test_data", "report_internal_error", "rand_byte", "bridge_loop_invariant_consume", "bridge_loop_invariant_produce", "bridge_loop_iteration_begin", "bridge_loop_iteration_end", "bridge_loop_iteration_assumptions", "nf_loop_iteration_begin", "nf_add_loop_iteration_assumptions", "nf_loop_iteration_end", "concretize_devices", "flow_consistency", "rte_eth_dev_count", "flood", "exit", "__uClibc_fini", "_stdio_term"};
+  std::regex symbol_re("klee*");
+  std::regex symbol2_re("_exit@plt*");
+
+  //Now we start iterating over the input trace and print only stuff we demarcate
+  int currently_demarcated = 0;
+  std::string currently_demarcated_fn = "";
+  std::string last_written_fn = "";
+  int call_gap = 1;
```

Comment:
Remove this too

Reply:
If i remove `call_gap` then effectively you are saying i remove the logging of "call to libVig" ? We have not tested these for other `nf`.